x-pack/plugins/enterprise_search/server/lib/route_config_helpers.test.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+const mockBuffer = {};
+jest.mock('@kbn/config-schema', () => ({
+  schema: {
+    buffer: () => mockBuffer,
+    object: () => ({}),
+  },
+}));
+
+const mockSchema = schema.object({});
+
+import { skipBodyValidation } from './route_config_helpers';
+
+describe('skipBodyValidation', () => {
+  it('adds body validation options to a route config, persisting previous properties like "path"', () => {
```

Comment:
For the first/main test title, I think it'd be clearer to focus on what the helper does add (`validate.body: schema.buffer()` and `options.body.parse.false` rather than what else is persisted - I think we could move that to the second test name

```suggestion
  it('adds `options.body.parse` and `validate.body` settings to a route config', () => {
```